src/wp-admin/includes/ajax-actions.php

```diff
@@ -231,15 +231,15 @@ function wp_ajax_wp_compression_test() {
 			if ( is_multisite() ) {
 				update_site_option( 'can_compress_scripts', 0 );
 			} else {
-				update_option( 'can_compress_scripts', 0, 'yes' );
+				update_option( 'can_compress_scripts', 0, 'on' );
```

Comment:
update_option accepts 'yes' or 'no' or true/false. Why does this PR use 'on' and 'off'?

Reply:
`on` and `off` are new additions in 6.6. See https://core.trac.wordpress.org/ticket/42441 / https://core.trac.wordpress.org/changeset/57920

Reply:
That's not exactly correct. `"on"` and `"off"` are the stored DB values, but the function should use boolean `true`/`false` values. For backwards compat, we still support `"yes"` and `"no"`.